docs/UserGuide.adoc

```diff
@@ -18,36 +18,37 @@ By: `Team AY1920S1-CS2103T-W11-3`      Since: `September 2019`      Licence: `MI
 
 == Introduction
 
-EatMe is for students who are trying to find *cheap and good places to eat or hang out*. EatMe will help you manage places you have been to, and keep track of your experience and expenditure. More importantly, EatMe is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, EatMe can get your tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+EatMe is a *food diary app for people who love good food*. It will help you manage places you have been to, keep track of your experience and expenditure, and even keep you up-to-date with the latest food trends and events!
+
+EatMe is optimized for those who like the speed of a Command Line Interface (CLI) while still maintaining the user-friendliness of a Graphical User Interface (GUI). *Interested to keep track of your favorite food places with maximum efficiency? Try EatMe today!* Jump to the <<Quick Start>> to get started.
 
 == Quick Start
 
-.  Ensure you have Java `11` or above installed in your Computer.
-.  Download the latest `eatme.jar` link:{repoURL}/releases[here].
-.  Copy the file to the folder you want to use as the home folder for the EatMe app.
-.  Double-click the file to start the app. The GUI should appear in a few seconds.
-+
+. Ensure you have Java `11` or above installed on your computer. If not, you can download it link:https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html[here].
+. Download the latest release of EatMe link:{repoURL}/releases[here].
+. Copy the file to the folder you want to use as the home folder for EatMe.
+. Double-click the file to start the app. The app should appear in a few seconds.
+
 image::Ui.png[width="790"]
-+
-.  Type the command in the command box and press kbd:[Enter] to execute it. +
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
 
-* **`add`**`\n Two Chefs Eating Place \a 116 Commonwealth Cres, #01-129 \c Chinese` : adds an eatery named Two Chefs Eating Place
-* **`review`**`\n Two Chefs Eating Place \d Nice Condensed Milk Pork Ribs \p 15 \r 4` : adds a review to Two Chefs Eating Place
-* *`random`* : shows a random eatery (if you can't decide what to eat)
-* *`bye`* : exits the app
+. Type a command in the command box and press kbd:[Enter] to execute it. +
+e.g. typing `help` and pressing kbd:[Enter] will open the help window.
+. Some example commands you can try:
+
+* `add \n Two Chefs Eating Place \a 116 Commonwealth Crescent, #01-129 \c Chinese` : Adds an eatery named Two Chefs Eating Place with the specified address and category.
+* `review \n Two Chefs Eating Place \d Nice Condensed Milk Pork Ribs \p 15 \r 4` : Adds a review to Two Chefs Eating Place with the specified description, price and rating.
```

Comment:
added date to reviews now. In the end \w 'date'

Reply:
Does the user need to know _why_ we use backslash?